### DISCUSSION TITLE
visual code clangd xmake 代码提示找不到头文件

### DISCUSSION THREAD

**Discussion by osenberg-x** [xmake-io/xmake#4107] [Q&A]
1. 已经生成并配置了 compile_commands.json, xmake 里面也已经进行了配置
2. 编译什么的都没问题
3. 就是在编辑界面下，clang 提示找不到头文件，看着很难受
![image](https://github.com/xmake-io/xmake/assets/42266214/98c1c7b2-f932-4fd5-ac28-89f4c5988d12)
![image](https://github.com/xmake-io/xmake/assets/42266214/44015ec7-e12c-4783-8866-ce4ac1952e78)
![image](https://github.com/xmake-io/xmake/assets/42266214/ddedc6a3-f6f6-49b7-aa83-2eb7e9d15540)

有什么办法解决吗

**Comment by waruqi** [comment]
https://xmake.io/#/zh-cn/plugin/more_plugins?id=clangd
